Header header
uint8 UniLocalIP0
uint8 UniLocalIP1
uint8 UniLocalIP2
uint8 UniLocalIP3
uint8 MAC0
uint8 MAC1
uint8 MAC2
uint8 MAC3
uint8 MAC4
uint8 MAC5
uint8 MultiRemoteIP0
uint8 MultiRemoteIP1
uint8 MultiRemoteIP2
uint8 MultiRemoteIP3
uint8 TxPortByte0
uint8 TxPortByte1
uint8 RxPortByte0
uint8 RxPortByte1
